scripts/process-csv-images

```diff
@@ -40,9 +40,16 @@ process_csv_images() {
     for image in "${csv_images[@]}"
     do
         echo "Processing image $image"
-
-        image_path="$(echo $image | grep / | cut -d/ -f2-)"
-        osbs_image_path="$(echo $image_path | sed 's/\//-/')"
+        # The below logic is specific to the amq-broker image
+        if [[ $image == *"amq-broker"* ]]; then
+            image_version="$(echo $image | cut -f2 -d":")"
+            group_image_version="$(echo ${image_version} | tr -d '.')"
+            major_version="$(echo $image_version | cut -f1 -d".")"
+            osbs_image_path="amq-broker-$major_version-amq-broker-$group_image_version-openshift:${image_version}"
+        else
+            image_path="$(echo $image | grep / | cut -d/ -f2-)"
+            osbs_image_path="$(echo $image_path | sed 's/\//-/')"
+        fi
```

Comment:
This is OK for now, but going forward need to think about how we can deal with these kind of things in a better way. We are likely to hit many issues like this as we add more products, so want to avoid all these specific paths in here as much as we can.